PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Application/Dtos/User/LoginDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace PrivateHospitals.Application.Dtos.User;
+
+public class LoginDto
+{
+    public string Email { get; set; }
+    public string Password { get; set; }
+}
```

Comment:
For some Dtos u use classes, but for other u use records. Please keep consistent

Reply:
Fixed